### PR TITLE
[2.0] CraftDataHandler method renames

### PR DESCRIPTION
**Pull request by AlexejheroYTB**
***
Changes made in this pull request:
* Renamed EditHarvestOutputList to EditHarvestable
* Renamed EditHarvestTypeList to EditHarvestType
* Renamed EditCookedCreatureList to EditCookable
* Added an EditItemSize overload (TechType, int, int)

--- Changes also done to the "redundant but user-friendly" methods

***

### CODE REVIEW

File: SMLHelper/Handlers/CraftDataHandler.cs [hunk 1]
```diff
@@ -88,6 +88,18 @@ public static void EditItemSize(TechType techType, Vector2int size)
...
         }
 
+        /// <summary>
+        /// <para>Allows you to edit item sizes for TechTypes.</para>
+        /// <para>Can be used for existing TechTypes too.</para>
+        /// </summary>
+        /// <param name="techType">The TechType whose item size you want to edit.</param>
+        /// <param name="x">The width of the item</param>
+        /// <param name="y">The hight of the item</param>
+        public static void EditItemSize(TechType techType, int x, int y)
```
Comment (PrimeSonic) [inline SMLHelper/Handlers/CraftDataHandler.cs]: Very nice. I like this. Very appropriate for a public API.

Comment (AlexejheroYTB) [reply SMLHelper/Handlers/CraftDataHandler.cs]: Just to be clear, X is the width and Y is the height, right?

File: SMLHelper/Handlers/CraftDataHandler.cs [hunk 2]
```diff
@@ -61,7 +61,7 @@ public static void EditHarvestOutputList(TechType techType, TechType harvestOutp
         /// </summary>
         /// <param name="techType">The TechType whose HarvestType you want to edit.</param>
         /// <param name="harvestType">The HarvestType for that TechType.</param>
-        public static void EditHarvestTypeList(TechType techType, HarvestType harvestType)
+        public static void EditHarvestType(TechType techType, HarvestType harvestType)
```
Comment (PrimeSonic) [inline SMLHelper/Handlers/CraftDataHandler.cs]: This is a good change. The original method name is informed too much by the underlying implementation.
The new method name is more straight to the point about what it intends to accomplish.

**Comment by AlexejheroYTB** [comment]
Fixed typos, ready to merge

File: SMLHelper/Handlers/CraftDataHandler.cs (hunk 1 above)
Comment (PrimeSonic) [reply SMLHelper/Handlers/CraftDataHandler.cs]: Wouldn't make sense otherwise

**Comment by PrimeSonic** [comment]
I'll go ahead and call this good here.

**Pull request merged**